update reference sn-react-typescript-boilerplate (#291)
SenseNet/sensenet.github.io: _docs/tutorials/how-to-deploy-to-netlify.md

## Git Repository Setup

> We recommend you to use our GitHub template that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-typescript-boilerplate). _If you don't know how to use a template repository check this [article](https://help.github.com/en/articles/creating-a-repository-from-a-template)_.

Reviewer: ```suggestion
We recommend you to use our starter template that you can find here [sn-react-typescript-boilerplate](https://github.com/SenseNet/sn-client/tree/master/examples/sn-react-typescript-boilerplate).
```

Author: fixed on 0682e100dc19cdb1fdbb633e6eca45d8e76c16e5

thanks
